specification/network/resource-manager/Microsoft.Network/stable/2022-09-01/azureFirewall.json

```diff
@@ -437,6 +437,80 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/azureFirewalls/{azureFirewallName}/packetCapture": {
```

Comment:
>packetCapture [](http://example.com/codeflow?start=135&length=13)

make it a verb capturePacket?

Reply:
Want to keep consistency between features and repos.